src/components/Input.jsx

```diff
@@ -0,0 +1,52 @@
+const Input = ({
+  id,
+  name = 'theme',
+  value,
+  checked,
+  style = 'duration-200 h-4 w-4 cursor-pointer appearance-none rounded-full transition-all',
+  className,
+}) => {
+  return (
+    <input
+      className={`${style} ${className}`}
```

Comment:
Dont you think this `style` prop is redendant? Why not add the content here directly instead? See below:

```suggestion
      className={`duration-200 h-4 w-4 cursor-pointer appearance-none rounded-full transition-all ${className}`}
```